other/materials_designer/passivate_nanoribbon.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>Let's move elsewhere - close to termination selection.</p><p> </p><p>We can also show the coordination number distribution plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>